ironfish-rust/src/errors.rs

```diff
@@ -23,7 +23,7 @@ pub enum SaplingKeyError {
 
 impl fmt::Display for SaplingKeyError {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        write!(f, "{}", self)
+        write!(f, "{:?}", self)
```

Comment:
__Note for others__:
{} causes the `fmt::Display` trait to be used, {?} causes the `fmt::Debug` trait to be used for rendering `self`

https://doc.rust-lang.org/std/fmt/index.html#fmtdisplay-vs-fmtdebug